packages/@aws-cdk/aws-eks/README.md

```diff
@@ -518,9 +518,35 @@ cluster.addCapacity('BottlerocketNodes', {
 
 To define only Bottlerocket capacity in your cluster, set `defaultCapacity` to `0` when you define the cluster as described above.
 
-Please note Bottlerocket does not allow to customize bootstrap options and `bootstrapOptions` properties is not supported when you create the `Bottlerocket` capacity. 
+Please note Bottlerocket does not allow to customize bootstrap options and `bootstrapOptions` properties is not supported when you create the `Bottlerocket` capacity.
 
+### Service Accounts
 
+With services account you can provide Kubernetes Pods access to AWS resources.
+
+```ts
+// add service account
+const serviceAccount = cluster.addServiceAccount('MyServiceAccount'));
```

Comment:
```suggestion
const serviceAccount = cluster.addServiceAccount('MyServiceAccount');
```

Reply:
Added a warning explaining the missing role conditions and the implication of this.